cmd/metrics-v2.go

```diff
@@ -502,7 +502,7 @@ func getS3RequestsCanceledMD() MetricDescription {
 		Subsystem: requestsSubsystem,
 		Name:      canceledTotal,
 		Help:      "Total number S3 requests that were canceled from the client while processing",
-		Type:      counterMetric,
+		Type:      gaugeMetric,
```

Comment:
Why a gauge?

Reply:
It won't show up otherwise if counter is '0' @kerneltime